app/helpers/link_migration_helper.rb

```diff
@@ -0,0 +1,15 @@
+module LinkMigrationHelper
```

Comment:
I might recommend testing this against production data if you haven't already, by pulling the prod db into staging, and/or into your dev environment, and running this against it. Could be the client is the best person to confirm it doesn't mess anything up, since they know the annotations best. Maybe don't deploy this PR to production before our meeting Friday, but have staging up to date with prod data, with this PR deployed and that data migration completed, before we meet, so he can confirm it's all good.

Reply:
Thanks Jamie! I did pull prod data into my dev environment and ran the migration there, and I'll do the same for staging.